App section

PovilasZubrys opened the pull request.

PovilasZubrys committed:
small structure changes

PovilasZubrys committed:
Merge branch 'develop' into App-section

PovilasZubrys committed:
small content changes

PovilasZubrys committed:
Content changes

PovilasZubrys committed:
Merge branch 'develop' into App-section

PovilasZubrys committed:
Merge branch 'develop' into App-section

PovilasZubrys committed:
Merge branch 'develop' into App-section

PovilasZubrys committed:
Content, structure and responsive design

PovilasZubrys committed:
Deleted app.css

PovilasZubrys committed:
Update @ custom.css

PovilasZubrys committed:
Merge branch 'develop' into App-section

PovilasZubrys committed:
Responsive changes

rositatisor reviewed (changes requested):
- Title reikėtų nuimti margin-left, kad tiek title, tiek p, tiek mygtukai lygiuotų kairėje pusėje;
**Pastebėjimas: 
Note: Only use one ``<h1>`` per page - this should represent the main heading/subject for the whole page.**
Reikia pakeisiti tag'ą iš h1, nes hero jau yra panaudotas h1, kuris vienintelis gali būti puslapyje. Tuomet turėtų susitvarkyti ir margin, nes dabar margin'o styling ateina nuo hero sekcijos.
- Mygtukuose neveikia Quicksand, sans-serif šriftas (nes nustatyta "Font Awesome 5 Brands") ir norėtųsi tarpelio tarp icon'ų ir teksto;

PovilasZubrys committed:
Fixes

Changed h1 to h4
Fixed font in buttons
+Added paddings to left and right columns to fit the design

Matulanas reviewed (changes requested):
- Right Column:

.mobile-1 klasei reiktų pakeisti **z-index** į **1**, kadangi turime tik **2** sluoksnių elementą.
                 kadangi dedame abiems sluoksniams **z-index**, tokiu atveju **position** parametras nereikalingas.
.mobile-2 reiktų uždėti **z-index -1**, kadangi turime **2** sluoksnių elementą ir šis elementas jo yra už jo. 

- Responsive:

Didžioji dalis kodo yra perrašoma su tais pačiais parametrais.
Reiktų palikti tik tas eilutes su klasėmis, kurių parametrus keiti pasikeičiant rezoliucijai. 

- Trūksta Buttons hoverio

PovilasZubrys committed:
Style changes and removed unnecessary code
